src/components/claiming/UploadFile.vue

```diff
@@ -21,19 +21,25 @@
         :http-request="handleUpload"
         :on-change="handleUpload"
         :limit="1"
+        :file-list="file"
         :on-exceed="handleExceed"
+        :on-remove="clearClaimingInfo"
       >
         <i class="el-icon-upload"></i>
-        <div class="el-upload__text">Suelta tu archivo aquí o <em>haz clic para cargar</em></div>
+        <div class="el-upload__text">Drag your file here or <em>click to upload</em></div>
         <div slot="tip" class="el-upload__tip">Only application/json files supported</div>
       </el-upload>
+      <p v-if="uploadFileError" class="error">
```

Comment:
If you upload a file once and you try to upload it again it shows an error. The expected behaviour is allow upload the file as many times as you want

Reply:
You need to delete the current file and upload it again. You cannot upload more than one.